lib/cadet_web/controllers/answer_controller.ex

```diff
@@ -3,6 +3,22 @@ defmodule CadetWeb.AnswerController do
 
   use PhoenixSwagger
 
+  alias Cadet.Assessments
+
+  def submit(conn, %{"questionid" => question_id, "answer" => answer}) do
+    case Assessments.answer_question(question_id, conn.assigns.current_user, answer) do
+      {:ok, _nil} ->
+        send_resp(conn, :ok, "OK")
+
+      {:error, {resp, message}} ->
```

Comment:
I'm using `{:error, {status, message}} ->`. I think we should standardise to that cos `status` for status code is clearer I think